File: weknot_android/app/src/main/java/com/example/weknot_android/room/entity/videocall/VideoCall.java
package com.example.weknot_android.room.entity.videocall;

import com.example.weknot_android.room.entity.user.User;

public class VideoCall {

    private User user;

    private Integer roomIdx;

    public VideoCall(User user, Integer roomIdx) {
        this.user = user;
        this.roomIdx = roomIdx;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public Integer getRoomIdx() {
        return roomIdx;
    }

    public void setRoomIdx(Integer roomIdx) {
        this.roomIdx = roomIdx;
    }
}

File: weknot_android/app/src/main/java/com/example/weknot_android/model/entity/videocall/VideoCall.kt
package com.example.weknot_android.model.entity.videocall

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.example.weknot_android.model.entity.user.User

@Entity(tableName = "video_call_table")
class VideoCall(@field:PrimaryKey
                var idx: Int,
                var user: String,
                var roomIdx: Int)